App crash 0.8.0-rc.1 on WoW Classic at Wago tab

Firstly, thank you for your work, love the software! 

I'm using 0.8.0-rc.1 and it seems to crash on WoW Classic version at Wago tab.

**Describe the bug**
Open Ajour, select Classic and click on Wago, the software crashes instantly, all the other are working ok on the Wago tab.

OS: Win 10
Ajour: 0.8.0-rc.1
Addon: WeakAura 3.2.1-classic

**Log Output**
Just the last bit, if you need more let me know:

11:56:58.399 [ajour_core::parse][DEBUG] Classic - 61 curse packages fetched
11:56:58.399 [ajour_core::parse][DEBUG] Classic - 0 tukui packages fetched
11:56:58.633 [ajour_core::parse][DEBUG] Classic - 22 wowi packages fetched
11:56:58.633 [ajour_core::parse][DEBUG] Classic - 0 townlong packages fetched
11:56:58.633 [ajour_core::parse][DEBUG] Classic - 0 git packages fetched
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 58 addons built from curse packages
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 0 addons built from tukui packages
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 0 addons built from wowi packages
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 0 addons built from townlong-yak packages
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 0 addons built from git packages
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 4 unknown addon folders
11:56:58.635 [ajour_core::parse][DEBUG] Classic - 62 addons successfully parsed
11:56:58.640 [ajour::gui::update][DEBUG] Message::ParsedAddons(Classic, 62 addons)
11:57:00.174 [ajour::gui::update][DEBUG] Interaction::ModeSelected(MyWeakAuras(Classic))
(crashes here)
11:57:00.186 [panic][ERROR] thread 'main' panicked at 'called `Option::unwrap()` on a `None` value': crates\weak_auras\src\lib.rs:520
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: DefSubclassProc
  22: DefSubclassProc
  23: CallWindowProcW
  24: DispatchMessageW
  25: SendMessageTimeoutW
  26: KiUserCallbackDispatcher
  27: NtUserDispatchMessage
  28: DispatchMessageW
  29: <unknown>
  30: <unknown>
  31: <unknown>
  32: <unknown>
  33: <unknown>
  34: <unknown>
  35: <unknown>
  36: <unknown>
  37: <unknown>
  38: <unknown>
  39: BaseThreadInitThunk
  40: RtlUserThreadStart

--- a/crates/weak_auras/src/lib.rs
+++ b/crates/weak_auras/src/lib.rs
@@ -148,6 +148,7 @@ async fn parse_weak_auras(
             a.status = AuraStatus::UpdateAvailable;
         }
     });
+    auras.retain(|a| !a.displays.is_empty());
 
     Ok(auras)
 }
@@ -277,6 +278,7 @@ async fn parse_platers(
             a.status = AuraStatus::UpdateAvailable;
         }
     });
+    auras.retain(|a| !a.displays.is_empty());
     auras.sort_by_key(|a| a.slug.clone());
     auras.dedup_by_key(|a| a.slug.clone());
 

